src/AccountManagement.ts

```diff
@@ -112,14 +110,14 @@ export class AccountManagement {
   /**
    * Send an invitation to an Account to join the signing Identity as a secondary Account
    *
-   * @note this will create an AuthorizationRequest which has to be accepted by the corresponding Account.
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create an {@link AuthorizationRequest Authorization Request} which has to be accepted by the `targetAccount`.
```

Comment:
```suggestion
   * @note this will create an {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `targetAccount`.
```